rcldotnet_common/Ret.cs

```diff
@@ -43,5 +43,13 @@ public enum RCLRet {
       WaitSetEmpty = 901,
       WaitSetFull = 902,
     }
+
+    public enum RMWRet {
```

Comment:
This should be renamed to `RMWNodeName` or something like that, these error codes are not the same as for the RMW layer, these are specific to `rmw_validate_node_name`

Reply:
I renamed to `RMWNodeNameRet`